nan.h

```diff
@@ -173,7 +173,7 @@ NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
 }
 
 /* io.js 1.0  */
-#if NODE_MODULE_VERSION >= IOJS_1_0_MODULE_VERSION \
+#if NODE_MODULE_VERSION >= NODE_0_12_MODULE_VERSION \
  || NODE_VERSION_AT_LEAST(0, 11, 15)
```

Comment:
Neither do I. It was added precisely for this. Tests even pass on Travis and AppVeyor too.

On February 20, 2015 10:26:35 AM EET, Ben Noordhuis <notifications@github.com> wrote:
>> @@ -173,7 +173,7 @@ NAN_INLINE v8::Local<T>
>_NanEnsureLocal(v8::Local<T> val) {
>>  }
>>  
>>  /* io.js 1.0  */
>> -#if NODE_MODULE_VERSION >= IOJS_1_0_MODULE_VERSION \
>> +#if NODE_MODULE_VERSION >= NODE_0_12_MODULE_VERSION \
>>   || NODE_VERSION_AT_LEAST(0, 11, 15)
>
>I don't understand why the `NODE_VERSION_AT_LEAST` doesn't work here.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rvagg/nan/pull/285/files#r25054541